resources/docker-compose-regtest.yml

```diff
@@ -77,9 +77,6 @@ services:
     networks:
       local:
         ipv4_address: 10.10.0.13
-    links:
-      - bitcoin
-      - electrs
     depends_on:
```

Comment:
Here it should depend on bitcoin too. Add - bitcoin here under the depends_on directive